src/libraries/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.SecKeyRef.macOS.cs

```diff
@@ -36,7 +37,7 @@ internal static partial class AppleCrypto
 
         internal static SafeSecKeyRefHandle ImportEphemeralKey(ReadOnlySpan<byte> keyBlob, bool hasPrivateKey)
         {
-            Debug.Assert(keyBlob != null);
+            Debug.Assert(Unsafe.IsNullRef(ref MemoryMarshal.GetReference(keyBlob)));
```

Comment:
This looks backward to me.

```suggestion
            Debug.Assert(!Unsafe.IsNullRef(ref MemoryMarshal.GetReference(keyBlob)));
```

Reply:
Oops, thanks!